examples/create-example.yaml

```diff
@@ -8,8 +8,10 @@ spec:
   clusterName: "my-cluster"
   baseUrl: "https://management.azure.com/"
   authBaseUrl: "https://login.microsoftonline.com"
-  tenantId: "REPLACE_WITH_TENANT_ID"
   azureCredentialSecret: "REPLACE_WITH_K8S_SECRETS_NAME"
+  privateCluster: false,
+  windowsAdminUsername: "my-admin-user",
+  windowsAdminPassword: "my-secure-passworD",
```

Comment:
If you want to include windows in this example then you can remove https://github.com/rancher/aks-operator/blob/master/examples/create-windows-example.yaml

For windows to work you need to set `networkPlugin`, `virtualNetwork`, and `subnet` and the name of the node pool is restricted to 6 characters, `workers2` is too long. This is why I had broken it out into a separate example.

Reply:
updated